azure-pipelines.yml

```diff
@@ -301,17 +301,13 @@ jobs:
 
 - job: BuildDockerImage
   displayName: Build Docker Image
-
-  dependsOn: ExtractMetadata
-  condition: succeeded()
   strategy:
     matrix:
-      AMD64:
-        pool: ${{ variables.ubuntu_pool }}
-        artifactName: docker-amd64
-      ARM64:
-        pool: ${{ variables.ubuntu_arm64_pool }}
-        artifactName: docker-arm64
+      ${{ each arch in parameters.architectures }}:
+        Alpine ${{ arch.name }}:
+          pool: ${{ arch.pool }}
+          artifactName: docker-${{ arch.value }}
+          dockerfile: Dockerfile
```

Comment:
Maybe renaming the `Dockerfile` to make it more specific, such as `alpine.dockerfile`, given we already have another base - Azure Linux.